print/ghostscript/Portfile

```diff
@@ -106,10 +107,14 @@ configure.cppflags-replace \
                     -I${prefix}/include \
                     -isystem${prefix}/include
 
-configure.args      --disable-compile-inits \
+configure.args      --enable-gpdl \
```

Comment:
Doesn't `--enable-gpdl` here conflict with `--without-gpdl` that you added a few lines down?

Reply:
Due to a quirk in how the Makefiles are written, both `--enable-gpdl` and `--with-gpdl` are required to build gpdl.